src/rpc/bootstrap_node_selector.h

```diff
@@ -83,7 +83,7 @@ namespace bootstrap_node
       std::string address;
       size_t fails;
 
-      void handle_result(bool success);
+      void handle_result(bool success, size_t score = 2);
```

Comment:
Just a curiosity: why 2?

Reply:
That's the existing number that got added in https://github.com/monero-project/monero/pull/6357

If a node has a lower height than our last checkpoint it should get more fail points than just 2, because it is useless to us.

Reply:
Oh, right! Thanks for the explanation.